std_msgs/Header header
float32 theta
float32 theta_dot
float32 phi
float32 phi_dot
